steam.py

```diff
@@ -217,14 +247,20 @@ def friday_phrase_randomizer():
         phrase = phrases[random_index]
         return(phrase)
 
+# Get entire info for games from wishlist.json. Helpful to call when building a reply string.
+def get_complete_info_for_game(gameid):
+    dict_info = {}
+    for appid in wishlist_json:
+        if gameid == int(appid):
+            dict_info.update(wishlist_json[f'{appid}'])
+    return dict_info
+
 def clear_special_characters(string_to_manulate):
+    special_characters = [u"\u2122", u"\u00AE"]

```

Comment:
Keep in mind these are characters we may want to be aware of in other places, for now since we only use it here it makes sense to keep it local to the function, but might want to move in the future, maybe when we create a config file or something similar